plugins/course-apps/proctoring/Settings.jsx

```diff
@@ -527,6 +525,7 @@ const ProctoringSettings = ({ onClose }) => {
           setSubmissionInProgress(false);
           setCourseStartDate(settingsResponse.data.course_start_date);
           setAvailableProctoringProviders(settingsResponse.data.available_proctoring_providers);
+          setAllowedProctoringEscalationEmail(settingsResponse.data.requires_escalation_email_providers);
```

Comment:
For consistency, could we call it `setRequiresEscalationEmailProviders` ?

Reply:
`setAllowedProctoringEscalationEmail` renamed to `setRequiresEscalationEmailProviders` for consistency.